geps/gep-1709.md

```diff
@@ -25,6 +25,17 @@ The following are items that we intend to resolve before we consider this GEP
 - We need to think about how we're going to highlight and report on the
   results for `experimental` features, and make sure this is explicitly
   written out.
+- We are [gathering feedback from SIG Arch][sig-arch-feedback]. We need to let
+  this soak for some time, bring up the topic at SIG Arch community meetings, and
+  generally make changes according to their feedback before considering this
+  `implementable`.
+- There are lingering concerns from several community members that having high
```

Comment:
The per group approach is the direction that SMI went down (see [here](https://meshery.io/service-mesh-interface)). I think a more granular approach has a lot of benefits for adoption

Reply:
Ok! There's been 3 people now that have pushed for this route, I think we might end up going that way. If you or anyone else wants to make a PR to update the GEP proposing the change you'd like to see, please do go for it. Otherwise I'll tackle this TODO as one of my upcoming PRs.